### ISSUE TITLE
Generate Release Notes extensions does not fail if it fails to locate the template file

### ISSUE THREAD

**Issue by wpschaub** [rfennell/AzurePipelines#740]
**Describe the bug**

We intentionally reconfigured the Generate Release Notes based on Release Comparison API to point at a non-existent template file as part of troubleshooting. The extension reports the error that it cannot find the template file as shown in log extract:

```
2020-06-13T05:39:08.8148725Z (node:5600) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: ENOENT: no such file or directory, open 'C:\DVDVAZDOAGENT01-1\_work\r95\a\ReleaseNotesTemplate.mk2'
2020-06-13T05:39:08.8148809Z End of payload data dump
2020-06-13T05:39:08.8149175Z ##[debug]Using template mode File
2020-06-13T05:39:08.8149232Z Loading template file C:\DVDVAZDOAGENT01-1\_work\r95\a\ReleaseNotesTemplate.mk2
2020-06-13T05:39:08.8244797Z ##[section]Finishing: Generate Release Notes based on Release Comparision API
```
The task, however, succeeds:
![image](https://user-images.githubusercontent.com/12176555/84561207-56bbaf00-acff-11ea-825e-5afe14f91795.png)

Which results in the Git base WIKI updater failing as no release notes were generated.

**To Reproduce**
1. Point extension to a non-existent file.
2. Run pipeline.

**Expected behavior**
Not finding the template file should be a fatal error and result in the extension failing in the pipeline.

**Logs**
All logs are saved and can be submitted on demand.

**What is the issue see:**
 - Azure DevOps Services (VSTS)
 - Azure DevOps Server (TFS) - Specify Version
 - Version 2.50.14

**Additional context**

**Comment by rfennell** [comment]
I have added a check to make sure the first is present, if it is not it will raise an Azure DevOps error.

I will publish it Monday evening, as I need a demo for my Usergroup session

**Comment by rfennell** [comment]
Released in 1.12.3
